designs/virtualization/4.5/Scheduling/Affinity/Affinity.md

```diff
@@ -0,0 +1,63 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Affinity
+
+Introducing Affinity as it appears in-page and the Edit modal.
+
+## in-page placement
+
+![Affinity field in the Scheduling section ](img/Affinity-0-0.jpg)
+The affinity field is placed in the Workload details page, Details tab, Scheduling section, Policies columns.
+
+![Affinity field in the Scheduling section ](img/Affinity-0-1.jpg)
+The affinity field counts the number of rules applied.
+
+## Affinity rules list
+
+![Affinity all rules modal ](img/Affinity-1-0.jpg)
```

Comment:
Rules with "Preferred" - with instead of will

Reply:
Should the title of this modal be "Affinity rules" or "affinity rules overview". It feels a bit odd to have two edit modals.

Reply:
Right, will change to "Overview".
"List" is too technical and obvious, right?

Reply:
I suppose it could be list but I prefer Overview :) But don't feel strongly.

Reply:
Overview it is